lib/vm/src/instance.rs

```diff
@@ -887,6 +894,7 @@ impl InstanceHandle {
     #[allow(clippy::too_many_arguments)]
     pub unsafe fn new(
         instance_ptr: NonNull<u8>,
+        owned_instance: bool,
```

Comment:
We can consider this always false, since it inits the `InstanceHandle`, so we don't really need to receive it

Reply:
It's a 2 step workflow, so I prefer to make it super explicit, in case it changes in the future.

Reply:
I don't think that should change in the future.

We can asume `InstanceHandle::new` will always be owned. Only generating it other way (like the `from_vmctx`can change that).
Either way, since right now is not really needed we can skip the new argument for now, add a comment and come back later if needed.

Reply:
Agree.